visitor.go

```diff
@@ -30,11 +30,16 @@ func DFSWalk(d *DAG, visitor Visitor) {
 		stack.Push(sv)
 	}
 
+	visited := make(map[string]bool, d.getSize())
```

Comment:
This modification fixes a problem with the previous version. When two vertices point to the same vertex, it may cause the destination vertex to be visited twice.